pages/index.js

```diff
@@ -2,126 +2,147 @@ import { useState } from 'react';
+              <option value="gassigehen">Gassi gehen</option>
+              <option value="sport">Sport</option>
+              <option value="otherthings">Sonstiges</option>
+            </select>
+          </div>
+        </section>
+        <section>
+          <h4>Altersgruppe</h4>
+          <div>
+            <div>
+              <label htmlFor="welpen">Welpen</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="welpen"
+                data-displayname="Welpen"
+              />
+            </div>
+            <div>
+              <label htmlFor="junghunde">Junghunde</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="junghunde"
+                data-displayname="Junghunde"
+              />
+            </div>
+            <div>
+              <label htmlFor="erwachsen">Erwachsen</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="erwachsen"
+                data-displayname="Erwachsen"
+              />
+            </div>
             <div>
-              <div>
-                <label htmlFor="welpen">Welpen</label>
-                <input
-                  type="checkbox"
-                  name="ageGroup"
-                  id="welpen"
-                  data-displayname="Welpen"
-                />
-              </div>
-              <div>
-                <label htmlFor="junghunde">Junghunde</label>
-                <input
-                  type="checkbox"
-                  name="ageGroup"
-                  id="junghunde"
-                  data-displayname="Junghunde"
-                />
-              </div>
-              <div>
-                <label htmlFor="erwachsen">Erwachsen</label>
-                <input
-                  type="checkbox"
-                  name="ageGroup"
-                  id="erwachsen"
-                  data-displayname="Erwachsen"
-                />
-              </div>
-              <div>
-                <label htmlFor="senior">Senior</label>
-                <input
-                  type="checkbox"
-                  name="ageGroup"
-                  id="senior"
-                  data-displayname="Senior"
-                />
-              </div>
+              <label htmlFor="senior">Senior</label>
+              <input
+                type="checkbox"
+                name="ageGroup"
+                id="senior"
+                data-displayname="Senior"
+              />
             </div>
           </div>
-        </div>
+        </section>
+        <section>
+          <div>
+            <label htmlFor="description">Beschreibung</label>
+            <textarea
+              name="description"
+              maxLength={300}
+              rows="4"
+              cols="50"
+              placeholder="Text - Beschreibung"
+            />
+          </div>
```

Comment:
what do you use these divs in your section? This seems like unnecessary nesting.